contents/csharp/IntroductionToCSharp.md

```diff
@@ -0,0 +1,148 @@
+<frontmatter>
+  title: Introduction to C#
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to C#
+
+Author(s:) [Yu Pei, Henry](https://github.com/YuPeiHenry)
+
+## What is C#
+
+C# is an Object-Oriented Programming Language on the .NET framework, and uses Garbage Collection for memory management.
+
+From Wikipedia:
+
+C# (pronounced C sharp) is a general-purpose, multi-paradigm programming language encompassing strong typing, lexically scoped, imperative, declarative, functional, generic, object-oriented (class-based), and component-oriented programming disciplines.
+It was developed around 2000 by Microsoft within its .NET initiative and later approved as a standard by Ecma (ECMA-334) and ISO (ISO/IEC 23270:2018). C# is one of the programming languages designed for the Common Language Infrastructure.
+
+## Why Learn C#
+
+C# is a relatively high level language and is not difficult to pick up (a little more difficult than Python and a little easier than Java.)
+
+### Cross platform support
+
+C# is built on the .NET framework and has cross platform support extended by the (Mono project)[https://www.mono-project.com/], and similarly the complete runtime implementation from open source (CoreCL)[https://github.com/dotnet/coreclr].
+C# is also used by the Unity Game Engine, which has high cross-platform support for game developers.
+
+### Popular Language
+
+From (Armina Mkhitaryan on Medium)[https://medium.com/sololearn/why-is-c-among-the-most-popular-programming-languages-in-the-world-ccf26824ffcb]
+
+Being powerful, flexible, and well-supported has meant C# has quickly become one of the most popular programming languages available.
```

Comment:
Being powerful, flexible, and well-supported ~~has meant~~ enabled C# to quickly become one of the most popular programming languages available.